tests/foreman/api/test_subscription.py

```diff
@@ -58,31 +57,38 @@ def rh_repo(module_org):
 
 
 @pytest.fixture(scope='module')
-def custom_repo(rh_repo, module_org):
+def custom_repo(rh_repo, module_sca_manifest_org):
     custom_repo = entities.Repository(
-        product=entities.Product(organization=module_org).create(),
+        product=entities.Product(organization=module_sca_manifest_org).create(),
     ).create()
     custom_repo.sync()
     return custom_repo
 
 
 @pytest.fixture(scope='module')
-def module_ak(module_org, rh_repo, custom_repo):
+def module_ak(module_sca_manifest_org, rh_repo, custom_repo):
     """rh_repo and custom_repo are included here to ensure their execution before the AK"""
     module_ak = entities.ActivationKey(
-        content_view=module_org.default_content_view,
+        content_view=module_sca_manifest_org.default_content_view,
         max_hosts=100,
-        organization=module_org,
-        environment=entities.LifecycleEnvironment(id=module_org.library.id),
+        organization=module_sca_manifest_org,
+        environment=entities.LifecycleEnvironment(id=module_sca_manifest_org.library.id),
         auto_attach=True,
     ).create()
     return module_ak
 
 
-@pytest.mark.skip_if_not_set('fake_manifest')
+@pytest.fixture(scope='function')
+def duplicate_manifest():
+    """Provides a function-scoped manifest that can be used alongside function_entitlement_manifest
+    when two manifests are required in a single test"""
+    with Manifester(manifest_category=settings.manifest.entitlement) as manifest:
+        yield manifest
+
```

Comment:
Would we want to put this with other fixtures, so it could be used in other scenarios?

Reply:
Possibly, but so far this is the only module I've encountered that has required a duplicate manifest. If I find another module that requires a duplicate function-scoped manifest, then I'll move this fixture definition to `pytest_fixtures/component/taxonomy.py`.

Reply:
This fixture can be used in other modules with changes coming in other test modules. Moving to taxonomy.py would bring better visibility for this fixture.